content/ember-v4-typescript-support-update.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Ember v4 TypeScript Support Update
+authors:
+  - chriskrycho
+date: 2022-01-25T00:00:00.000Z
+tags:
+  - '2022'
+  - typescript
+
+---
+
+The [Typed Ember](https://github.com/typed-ember) team—[Chris Krycho (@chriskrycho)](https://github.com/chriskrycho), [Dan Freeman (@dfreeman)](https://github.com/dfreeman), and [James Davis (@jamescdavis)](https://github.com/jamescdavis)—is happy to announce that the `@types` type definition packages, maintained on [Definitely Typed](https://github.com/DefinitelyTyped/DefinitelyTyped), now have full support for Ember 4.x! This is *two major changes* with *two big benefits*.
+
+The two major changes:
+
+* **All APIs which were deprecated during the 3.x cycle and removed in Ember v4.0.0 have been dropped from the types.** (Amusingly, this includes a number of things which were deprecated in 2.x and removed at 3.0, and even a few which were deprecated the 1.x era and removed at 2.0!) Additionally, a number of types representing private API have been removed, in line with our standing policy of providing types for *only* public APIs.
+
+* **The minimum supported TypeScript version is 4.4.** This aligns with the direction we expect to propose for Ember’s own TypeScript support policy, where major releases will generally adopt a recent TypeScript version as their minimum supported version, and with a support policy similar to that for browsers and Node LTS releases. (Keep your eyes open for a forthcoming RFC tackling those details!)
```

Comment:
I thought "This aligns with the direction we expect to propose for Ember’s own TypeScript support policy" was hedging a bit too much. Does something like my edit still make sense?

```suggestion
* **The minimum supported TypeScript version is 4.4.** This aligns with our upcoming for Ember’s own TypeScript support policy, where major releases will generally adopt a recent TypeScript version as their minimum supported version, and with a support policy similar to that for browsers and Node LTS releases. Keep your eyes open for a forthcoming RFC tackling those details!
```

Reply:
```suggestion
* **The minimum supported TypeScript version is 4.4.** This aligns with our upcoming proposal for Ember’s own TypeScript support policy, where major releases will generally adopt a recent TypeScript version as their minimum supported version, and with a support policy similar to that for browsers and Node LTS releases. (Keep your eyes open for a forthcoming RFC tackling those details!)
```